source/unified-test-format/unified-test-format.rst

```diff
@@ -1779,6 +1779,54 @@ The following arguments are supported:
 
 - ``session``: Optional string. See `commonOptions_session`_.
 
+runCursorCommand
+~~~~~~~~~~~~~~~~
+
+`Generic cursor returning command runner <../run-command/run-command.rst>.`__
+
+This method does not inherit a read concern or write concern (per the
+`Read and Write Concern <../read-write-concern/read-write-concern.rst#generic-command-method>`__
+spec), nor does it inherit a read preference (per the
+`Server Selection <../server-selection/server-selection.rst#use-of-read-preferences-with-commands>`__
+spec); however, they may be specified as arguments.
+
+This operation proxies the databases's ``runCursorCommand`` method and supports the same arguments and options.
+
+When executing the provided command, the test runner MUST fully iterate the cursor.
+This will ensure consistent behavior between drivers that eagerly create a server-side cursor and those that do so lazily when iteration begins.
+
+The following arguments are supported:
+
+- ``command``: Required document. The command to be executed.
+
+- ``commandName``: Required string. The name of the command to run. This is used
+  by languages that are unable preserve the order of keys in the ``command``
+  argument when parsing YAML/JSON.
+
+- ``readPreference``: Optional object. See `commonOptions_readPreference`_.
+
+- ``session``: Optional string. See `commonOptions_session`_.
+
+- ``batchSize``: Optional int32. Use this value to configure the batchSize sent on the getMore command.
+
+- ``maxTimeMS``: Optional int32. Use this value to configure the maxTimeMS sent on the getMore command.
```

Comment:
```suggestion
- ``maxTimeMS``: Optional int32. Use this value to configure the ``maxTimeMS`` option sent on subsequent ``getMore`` commands.
```

[getMore: Command Fields](https://www.mongodb.com/docs/manual/reference/command/getMore/#command-fields) only documents this as a non-negative integer. Same comment as above for `batchSize`.

Reply:
Followed up on this in https://github.com/mongodb/specifications/pull/1412/files#r1190733860.